cpp/log/cert.h

```diff
@@ -272,6 +277,26 @@ class TbsCertificate {
   DISALLOW_COPY_AND_ASSIGN(TbsCertificate);
 };
 
+inline std::ostream& operator<<(std::ostream& out,
+                                const Cert::Status& status) {
+  switch (status) {
+    case Cert::Status::TRUE:
+      out << "TRUE";
+      break;
+    case Cert::Status::FALSE:
+      out << "FALSE";
+      break;
+    case Cert::Status::ERROR:
+      out << "ERROR";
+      break;
+    case Cert::Status::UNSUPPORTED_ALGORITHM:
+      out << "UNSUPPORTED_ALGORITHM";
+      break;
```

Comment:
Hmm, never thought of that... I did it different from my usual, which is to `return` in every case (instead of `break`), and have a default behaviour (output something like "invalid Cert::Status", say) after the `switch` statement (I avoid `default` to get the compiler to warn us if new values get added to the `enum`).